aibolit/patterns/instanceof/instance_of.py

```diff
@@ -15,16 +15,32 @@ def __traverse_node(self, node):
         List of code lines
         """
         lines = []
-        for _, node_elem in node.filter(javalang.tree.BinaryOperation):
-            if node_elem.operator == 'instanceof' and node_elem.operandl.position is not None:
-                code_line = node_elem.operandl.position.line or node_elem.operandr.position.line
-                lines.append(code_line)
-        for _, node_elem in node.filter(javalang.tree.MethodInvocation):
-            if node_elem.member == 'isInstance':
-                lines.append(node_elem.position.line)
-
+        depth = 0
```

Comment:
You could move `_traverse_node` to `value` function.

Reply:
ok